### PR TITLE
rpy2 v3.3.5 (fix)

### PR DESCRIPTION
**Pull request by pauleve**
Auto-update is failing since 3.3.3 due to the patch merged upstream.

This PR updates to 3.3.5 by dropping the patch.

Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)

### CODE REVIEW

**Comment by pauleve** [comment]
There is now a test failure 
```
 rpy2.robjects.packages.PackageNotInstalledError: The R package "stats" is not installed
```

File: recipe/meta.yaml [hunk 1]
```diff
@@ -43,6 +43,7 @@ test:
   requires:
     - r-survival
     - r-ggplot2 >=2.2.1
+    - r-stats
```
Comment (pauleve) [inline recipe/meta.yaml:46]: The "stats" package is part of the r-base distribution, if I'm not mistaken; so there is no `r-stats` package..

Comment (ocefpaf) [reply recipe/meta.yaml:46]: Can you remove it and see how it goes?

**Comment by dbast** [comment]
Done via https://github.com/conda-forge/rpy2-feedstock/pull/70

**Pull request closed without merging**